chrome/content/test/tests.js

```diff
@@ -3,43 +3,7 @@
  * It uses the Jasmine JS testing library.
```

Comment:
It is clearer organization to have scripts named for the file they are unit testing. If they are integration tests then they can test multiple files.